src/server/scripts/Kalimdor/CavernsOfTime/EscapeFromDurnholdeKeep/boss_captain_skarloc.cpp

```diff
@@ -108,27 +110,31 @@ struct boss_captain_skarloc : public BossAI
         if (type != ESCORT_MOTION_TYPE)
             return;
 
-        // Xinef: we can rely here on internal counting
-        if (id == 1)
+        if (!_SpawnedAdds)
         {
```

Comment:
very nerdy comment, but variables in C++ are camelCase and methods are PascalCase